options/locale/locale_ja-JP.ini

```diff
@@ -2338,6 +2338,7 @@ team_desc_helper=チームの目的や役割を説明します。
 team_access_desc=リポジトリアクセス
 team_permission_desc=権限
 team_unit_desc=リポジトリのセクションへのアクセスを許可
+team_unit_helper=以下のアクセス制御ポリシーは公開リポジトリに対して効果しないです。
```

Comment:
You don't need to update this file. It's generated from `locale_en-US.ini` on https://crowdin.com/project/gitea.